### ISSUE TITLE
"Script error" with no other indication in the source code or output

### ISSUE THREAD

**Issue by alan-knight** [dart-lang/dart-pad#705]
https://dartpad.dartlang.org/b7502403dcb205a1fe2a

I'm sure there's some trivial error in the code, but it's not helping me find it.

**Comment by devoncarew** [comment]
I'm seeing this in devtools:

> Uncaught RangeError: Maximum call stack size exceeded

Not sure what we can do about the stack overflow exception here, but we should do a better job of drawing the user's attention to it. Probably minimally, switching to the console tab.

Thanks for the report!

**Comment by alan-knight** [comment]
Yeah, it was a stack overflow. Is there a way to get to devtools and debug it. Or even print the error message from the script? If it had said "stack overflow" instead of just "script error" that would have been enough.

**Comment by devoncarew** [comment]
I think this guy would help out a bit: https://github.com/dart-lang/dart-pad/pull/554, but I don't think I have time to finish it off.

**Comment by jcollins-g** [comment]
This case seems to have gotten worse since we last looked at it; there is now no error message or console output displayed at all.
